README.md

```diff
@@ -330,6 +330,25 @@ a.versions.size                           # 3
 a.versions.last.event                     # 'update'
 ```
 
+You can also use the corresponding callback-methods seperately instead of using
+the :on option. If you choose to use the callback-methods, PaperTrail will only
+track the according events - so `paper_trail_on_create` is basically the same as
+`has_paper_trail :on => :create`.
```

Comment:
Added. Since I am not an english native-speaker, every suggestion to my readme addition is much appreciated.